docs/docs/migrate-data.md

```diff
@@ -59,7 +59,7 @@ export CORAL_SITE_ID=""
 
 # This importer tool is designed to work with Coral at the following migration
 # version. This is the newest file in the
-# https://github.com/coralproject/talk/tree/develop/src/core/server/services/migrate/migrations
```

Comment:
I wonder why this was pointing at `develop` previously... oh well, targeting `main` is probably the right call though!